gradle-sls-packaging/build.gradle

```diff
@@ -85,3 +85,8 @@ publishPlugins.onlyIf {
 test {
     maxHeapSize = '1g'
 }
+
+// Configure the publishPlugins task
+tasks.publish.dependsOn publishPlugins
+project.ext.'gradle.publish.key' = System.env["GRADLE_KEY"]
+project.ext.'gradle.publish.secret' = System.env["GRADLE_SECRET"]
```

Comment:
tested locally that this gives me a different error message (unauthorized) if I set the expected environment variables than if I don't (missing publishing keys), so should be g2g!